config/vufind/Notifications.yaml

```diff
@@ -0,0 +1,24 @@
+# Configuration file for notifications
+
+Notifications:
+  # Activated languages.
+  # Add more as two character abbreviations.
+  languages:
```

Comment:
Might it make sense for this setting to be optional, and default to whatever languages are activated in config.ini? That way, the user can narrow down the language list if they want to, but by default, they won't be forced to configure their language list in two different places.